platform/qt/src/qmapboxgl.cpp

```diff
@@ -348,6 +481,13 @@ void QMapboxGL::setLatitude(double latitude_)
     d_ptr->mapObj->setLatLng(mbgl::LatLng { latitude_, longitude() }, d_ptr->margins);
 }
 
+/*!
+    \property QMapboxGL::longitude
+    \brief the map current longitude in degrees.
+
+    Setting a longitude doesn't necessarily mean it will be accepted since QMapboxGL
```

Comment:
What happens if the map rejects the longitude? Does it move to a constrained longitude, or to a wrapped longitude, or does the map refuse to move?